Sources/ComposableArchitecture/Store.swift

```diff
@@ -338,9 +308,13 @@ public final class Store<State, Action> {
       reducer: .init { localState, localAction, _ in
         isSending = true
         defer { isSending = false }
-        self.send(fromLocalAction(localAction), file: file, line: line)
+        let task = self.send(fromLocalAction(localAction), file: file, line: line)
```

Comment:
@krzysztofzablocki added instrumentation around this one in #23

Reply:
@diederich I'm not seeing the instrumentation around this?

Reply:
That was a little confusing. I didn't see it here either, but I had it locally. Re-did the merge and should be in now.